src/content/docs/fr/guides/cms/hashnode.mdx

```diff
@@ -0,0 +1,352 @@
+---
+title: Hashnode & Astro
+description: Ajouter du contenu à votre projet Astro en utilisant Hashnode comme CMS
+type: cms
+stub: false
+service: Hashnode
+i18nReady: true
+---
+
+import { FileTree } from '@astrojs/starlight/components';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import { Steps } from '@astrojs/starlight/components';
+
+[Hashnode] (https://hashnode.com/) est un CMS hébergé qui vous permet de créer un blog ou une simple publication.
+
+## Intégration avec Astro
+L'API [Hashnode Public API](https://apidocs.hashnode.com/) est une API GraphQL qui vous permet d'interagir avec Hashnode. Ce guide utilise [`graphql-request`](https://github.com/jasonkuhrt/graphql-request), un client GraphQL minimal qui fonctionne bien avec Astro, pour apporter vos données Hashnode dans votre projet Astro.
+
+### Prérequis
+Pour commencer, vous devez disposer des éléments suivants :
+
+1. **Un projet Astro** - Si vous n'avez pas encore de projet Astro, notre [Guide d'installation](/fr/install-and-setup/) vous permettra d'être opérationnel en un rien de temps.
+
+2. **Un site Hashnode** - Vous pouvez créer un site personnel gratuit en visitant [Hashnode](https://hashnode.com/).
+
+### Installation des dépendances
+
+Installez le paquet `graphql-request` en utilisant le gestionnaire de paquets de votre choix :
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install graphql-request
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm add graphql-request
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add graphql-request
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+## Créer un blog avec Astro et Hashnode
+
+Ce guide utilise [`graphql-request`](https://github.com/jasonkuhrt/graphql-request), un client GraphQL minimal qui fonctionne bien avec Astro, pour apporter vos données Hashnode dans votre projet Astro.
+
+### Prérequis
+
+1. Un Hashnode Blog
+2. Un projet Astro intégré avec le paquetage [graphql-request](https://github.com/jasonkuhrt/graphql-request) installé.
+
+Cet exemple va créer une page d'index qui liste les articles avec des liens vers des pages individuelles générées dynamiquement.
+
+### Récupération des données
+
+<Steps>
+1. Pour récupérer les données de votre site avec le paquet `graphql-request`, créez un répertoire `src/lib` et créez deux nouveaux fichiers `client.ts` & `schema.ts` :
+    <FileTree title="Project Structure">
```

Comment:
The `title` doesn't seem to be used (or I looked wrong) but in case of change it's best to translate it.
```suggestion
    <FileTree title="Structure du projet">
```